lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -61,7 +61,7 @@ def line2d_seg_dist(p1, p2, p0):
     x01 = np.asarray(p0[0]) - p1[0]
     y01 = np.asarray(p0[1]) - p1[1]
 
-    u = (x01*x21 + y01*y21)/float(abs(x21**2 + y21**2))
+    u = (x01*x21 + y01*y21) / abs(x21**2 + y21**2)
```

Comment:
Very genuine comment: are `x21` and `y21` complex or real numbers? Because in the latter case, what is the point of `abs(...)` then?

Reply:
Pretty sure they're intended to be reals (it's computing a 3D distance...).  Removed the call to abs, good catch.